microsoft/testsuites/kdump/kdumpcrash.py

```diff
@@ -265,7 +265,23 @@ def _kdump_test(self, node: Node, log_path: Path, log: Logger) -> None:
         free = node.tools[Free]
         total_memory = free.get_total_memory()
 
-        if "T" in total_memory and float(total_memory.strip("T")) > 1:
+        # Ubuntu Redhat and Suse have differnt proposed crashkernel settings
```

Comment:
Ubuntu, Redhat and Suse have different proposed crashkernel settings

Reply:
can we put guidance links here?

Reply:
Has added.